### PR TITLE
docs: fix invalid urls

### PR DESCRIPTION
**Pull request by WeiZhang555**
Fixes: #391

Replace invalid urls with new legal ones.

Signed-off-by: Zhang Wei <zhangwei555@huawei.com>

Carry https://github.com/kata-containers/runtime/pull/392

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [![Build Status](https://travis-ci.org/kata-containers/runtime.svg?branch=master)](https://travis-ci.org/kata-containers/runtime)
 [![Build Status](http://kata-jenkins-ci.westus2.cloudapp.azure.com/job/kata-containers-runtime-ubuntu-16-04-master/badge/icon)](http://kata-jenkins-ci.westus2.cloudapp.azure.com/job/kata-containers-runtime-ubuntu-16-04-master/)
+[![Build Status](http://jenkins.katacontainers.io/job/kata-containers-runtime-ubuntu-16-04-master/badge/icon)](http://jenkins.katacontainers.io/job/kata-containers-runtime-ubuntu-16-04-master/)
```
Comment (jodh-intel) [inline README.md]: Nice! Whoever set this up - thanks! :)

But it looks like you forgot to remove the old URL?

**Comment by jodh-intel** [comment]
Given the fact the entire CI is currently broken and we need it working asap, let's force-merge this doc-only change...

**Comment by WeiZhang555** [comment]
It's still not working, driving me crazy ...💢

**Comment by WeiZhang555** [comment]
@jodh-intel Yeah, it's back again! 🎆

**Pull request merged**